<?xml version="1.0"?>
<launch>

  <arg name="robot_namespace" default="bvr_SIM"/>
  <arg name="arm_namespace" default="main_arm_SIM"/> 
  <arg name="sim_suffix" default="_SIM"/>
  <arg name="VIO_enabled" default="false"/>
  <arg name="tf_prefix" default=""/>
  <arg name="kinematics_config" default="$(find robowork_ur_launch)/etc/bvr_main_arm_ex-ur5e-0_calibration.yaml"/>
  <arg name="hosting_robot_prefix" default="bvr_SIM/"/> 
  <arg name="ur5_e_robot_prefix" default="main_arm_SIM/"/> 

  <arg name="debug" default="false" />

  <rosparam command="load" file="$(find boat_planner)/config/points.yaml"/>

  <node name="boat_planner" pkg="boat_planner" type="boat_planner_node" output="screen">
    <param name="robot_namespace" value="$(arg robot_namespace)" />
    <param name="arm_namespace" value="$(arg arm_namespace)" />
    <param name="sim_suffix" value="$(arg sim_suffix)" />

    <!-- 
    <param name="allow_trajectory_execution" value="$(arg allow_trajectory_execution)"/>
    <param name="max_safe_path_cost" value="$(arg max_safe_path_cost)"/>
    <param name="jiggle_fraction" value="$(arg jiggle_fraction)" />
    <param name="max_sampling_attempts" value="$(arg max_sampling_attempts)" />
    -->
  </node>
  
</launch>